portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportTemplate.testcase

```diff
@@ -70,6 +71,7 @@ definition {
 			exportTemplateName = "Export Template Name");
 	}
 
+	@description = "Verify is possible export with relaunched process."
```

Comment:
I think this is the same meaning with the original one
`Verify is possible export a template with relaunched process`
We need to emphasize relaunch behavior instead of emphasizing export behavior.
e.g. @description = "Verify is possible relaunch the exported process."

Reply:
@JoyceWang08211 It really wasn't correct. But I did it the way you explained.